xbmc/pvr/dialogs/GUIDialogPVRChannelManager.cpp

```diff
@@ -1028,6 +1032,8 @@ bool IsItemChanged(const std::shared_ptr<CFileItem>& item)
   const std::shared_ptr<CPVRChannel> channel = member->Channel();
 
   return item->GetProperty(PROPERTY_CHANNEL_ENABLED).asBoolean() == channel->IsHidden() ||
+         item->GetProperty(PROPERTY_CHANNEL_CUSTOM_ENABLED).asBoolean() !=
```

Comment:
Shouldn't this read `==` ? ENABLED vs Hidden needs inverted logic; see line above this line.

Reply:
The IsUserSetHidden flag doesn't use inverted logic like "Active" is to "Hidden".  I see the concern, though. I think PROPERTY_CHANNEL_CUSTOM_ENABLED was just a horrible name to use.  What about renaming that to PROPERTY_CHANNEL_USER_SET_HIDDEN or CUSTOM_HIDDEN?

Reply:
> What about renaming that to PROPERTY_CHANNEL_USER_SET_HIDDEN or CUSTOM_HIDDEN?

Yes, that seems to fit better.

Reply:
Changed to **PROPERTY_CHANNEL_USER_SET_HIDDEN**.  Thanks again, that was a really bad name choice on my part.